core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/provision/ClusterCreationActions.java

```diff
@@ -651,6 +657,9 @@ protected void doExecute(ClusterCreationViewContext context, ClusterProxyGateway
                 jobService.schedule(context.getStackId(), StackJobAdapter.class);
                 syncJobService.schedule(context.getStackId(), StructuredSynchronizerJobAdapter.class);
                 aimJobService.schedule(context.getStackId());
+                if (CloudPlatform.MOCK.equalsIgnoreCase(context.getStack().cloudPlatform())) {
+                    existingStackPatcherJobService.schedule(context.getStackId(), StackPatchType.MOCK);
```

Comment:
Why should we hardcode this schedule? This way we bypass the testing of the patch scheduling.

Reply:
The reason was included in the commit message:

> As stack pathes usually only apply to already existing stacks, a workaround was added to schedule mock stack patchers for each mock stack after creation.